tests/oauth2/test_service_account.py

```diff
@@ -811,18 +812,19 @@ def test_refresh_iam_flow_non_gdu(self, call_iam_generate_id_token_endpoint):
         )
         request = mock.Mock()
         credentials.refresh(request)
-        req, iam_endpoint, signer_email, target_audience, access_token = call_iam_generate_id_token_endpoint.call_args[
+        req, iam_endpoint, signer_email, target_audience, access_token, universe_domain = call_iam_generate_id_token_endpoint.call_args[
```

Comment:
This is another example of how you can test the iam endpoints that are built. You can use this pattern instead of creating a separate method get_iam_sign_endpoint and testing it in impersonated_credentials